doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='business-central-settings-changing-permissions-proc_{context}']
+= Changing permissions for groups and roles in {CENTRAL}
+
+In {CENTRAL}, you cannot change permissions for an individual user. However, you can change permissions for groups and roles. The changed permissions apply to users that have the role or belong to the group that you changed.
+
+[NOTE]
+====
+Any changes you make to roles or groups affect all of the users associated with that role or group.
+====
+
+.Procedure
```

Comment:
Any prereqs? You are logged in to BC? Do I have to have the admin role?

Reply:
Added the Prerequisites.